datumaro/util/definitions.py

```diff
@@ -2,8 +2,16 @@
 #
 # SPDX-License-Identifier: MIT
 
+import os
+import os.path as osp
 from typing import Tuple
 
 DEFAULT_SUBSET_NAME = "default"
 BboxIntCoords = Tuple[int, int, int, int]  # (x, y, w, h)
 SUBSET_NAME_BLACKLIST = {"labels", "images", "annotations", "instances"}
+
+_CACHE_DIR = os.getenv("XDG_CACHE_HOME", osp.join(osp.expanduser("~"), ".cache"))
```

Comment:
```suggestion
_CACHE_DIR = osp.expanduser(os.getenv("XDG_CACHE_HOME", osp.join("~", ".cache")))
```

It would be better to expand `~` at the end as one might explicitly specify `~` in `XDG_CACHE_HOME`.